lib/metadataTypes/MetadataType.js

```diff
@@ -652,44 +622,22 @@ class MetadataType {
      * @returns {Promise<{metadata:MetadataTypeMap,type:string}>} Promise of item map
      */
     static async retrieveREST(retrieveDir, uri, overrideType, templateVariables) {
-        const options = {
-            uri: uri,
-            headers: {},
-        };
-        let moreResults;
-        let lastPage = null;
-        let results = {};
-        do {
-            options.uri = this.paginate(options.uri, lastPage);
-            let response;
-            await Util.retryOnError(`Retrying ${this.definition.type}`, async () => {
-                response = await this.client.RestClient.get(options);
-            });
-            const metadata = this.parseResponseBody(response.body);
-            results = Object.assign(results, metadata);
-            if (
-                this.definition.restPagination &&
-                Object.keys(metadata).length > 0 &&
-                response.body.page * response.body.pageSize < response.body.count
-            ) {
-                lastPage = Number(response.body.page);
-                moreResults = true;
-            } else {
-                moreResults = false;
-            }
-        } while (moreResults);
+        let response;
+        await Util.retryOnError(`Retrying ${this.definition.type}`, async () => {
+            response = this.definition.paginate
```

Comment:
I think this was renamed some time back to `this.definition.restPagination`

- [ ] please check variable name

Reply:
actually, compare deleted line 671 for the same